megamek/src/megamek/common/MechView.java

```diff
@@ -1184,12 +1212,16 @@ private static class LabeledElement implements ViewElement {
         
         @Override
         public String toPlainText() {
-            return label + ": " + value + "\n";
+            String htmlCleanedText = value.replaceAll("<[Bb][Rr]> *", "\n")
```

Comment:
Is that really all the html tags that can show up in this text?

Reply:
Ok for now. If we wind up having html elements that aren't getting caught then we can look into a more robust solution.